subworkflows/local/bam_variant_calling_germline_all/main.nf

```diff
@@ -223,11 +217,29 @@ workflow BAM_VARIANT_CALLING_GERMLINE_ALL {
                 known_sites_snps_tbi,
                 known_snps_vqsr)
 
+            // TO-DO: Should this VCF from the sentieon haploTYPER really be put into a variable called vcf_haploTYPECALLER
```

Comment:
@maxulysse @FriederikeHanssen : Any thoughts on that?  I think it should be renamed to `vcf_haplotyper` and then that should be added to the annotation over in `sarek.nf`.

Reply:
yeah maybe keeping them separate is a good idea. That way you could run both if you really wanted to

Reply:
I'll replace "my" usage of `vcf_haplotyper`with `vcf_sentieon_haplotyper`. This also seems to solve the problem of the unfiltered vcf ending up in the csv.